core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.scalar.db.transaction.consensuscommit;
+
+import com.scalar.db.api.DistributedTransactionAdmin;
+import com.scalar.db.api.DistributedTransactionFactory;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.api.TwoPhaseCommitTransactionManager;
+import com.scalar.db.config.DatabaseConfig;
+
+public class ConsensusCommitFactory implements DistributedTransactionFactory {
```

Comment:
This is the implementation of `DistributedTransactionFactory ` for Consensus Commit.